keepassxc-protocol.md

```diff
@@ -204,7 +204,8 @@ Response message data (success, decrypted):
     {
         "login": "user2",
         "name": "user2",
-        "password": "passwd2"
+        "password": "passwd2",
+        "expired": "true"
```

Comment:
Shouldn't this be "bool" since it can be true or false.

Reply:
Maybe, but I wanted to maintain the compatibility with other values. The success property also sets this as a string.

Reply:
Yah after viewing the rest of the protocol document, makes sense.